cluster.go

```diff
@@ -914,7 +914,7 @@ func (c *Cluster) open() error {
 			return fmt.Errorf("sending restart NodeJoin: %v", err)
 		}
 
-		c.Logger.Printf("wait for joining to complete")
+		c.Logger.Printf("%v wait for joining to complete", c.Node.ID)
```

Comment:
was trying to debug a deadlock... I think this log is slightly more helpful though